### ISSUE TITLE
Project is not loading

### ISSUE THREAD

**Issue by viktor-ferenczi** [FalsePattern/ZigBrains#61]
OS: Windows 11 64-bit, x86
IDE: CLion 2024.2

Configured zig and zls binaries properly. Both of them work.

Attempted to create a new Zig executable project, but the Project pane never loaded. It stays in "loading..." state.

Got error in CLion's log, reported to came from the ZigBrains plugin:
```
java.lang.RuntimeException: Cannot invoke (class=LocalHistoryBulkFileListener, method=after, topic=BulkFileListener)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeListener(MessageBusImpl.kt:699)
	at com.intellij.util.messages.impl.MessageBusImplKt.executeOrAddToQueue(MessageBusImpl.kt:514)
	at com.intellij.util.messages.impl.ToDirectChildrenMessagePublisher.publish$intellij_platform_core(CompositeMessageBus.kt:279)
	at com.intellij.util.messages.impl.MessagePublisher.invoke(MessageBusImpl.kt:471)
	at jdk.proxy2/jdk.proxy2.$Proxy90.after(Unknown Source)
	at com.intellij.openapi.vfs.newvfs.persistent.PersistentFSImpl.lambda$fireAfterEvents$20(PersistentFSImpl.java:1495)
	at com.intellij.openapi.vfs.newvfs.persistent.PersistentFSImpl.runSuppressing(PersistentFSImpl.java:1125)
	at com.intellij.openapi.vfs.newvfs.persistent.PersistentFSImpl.fireAfterEvents(PersistentFSImpl.java:1492)
	at com.intellij.openapi.vfs.newvfs.persistent.PersistentFSImpl.lambda$processEvent$8(PersistentFSImpl.java:1091)
	at com.intellij.openapi.vfs.newvfs.persistent.PersistentFSImpl.runSuppressing(PersistentFSImpl.java:1125)
	at com.intellij.openapi.vfs.newvfs.persistent.PersistentFSImpl.processEvent(PersistentFSImpl.java:1088)
	at com.intellij.openapi.vfs.newvfs.persistent.PersistentFSImpl.createChildFile(PersistentFSImpl.java:745)
	at com.intellij.openapi.vfs.newvfs.impl.VirtualFileSystemEntry.createChildData(VirtualFileSystemEntry.java:306)
	at com.falsepattern.zigbrains.project.ide.newproject.ZigProjectConfigurationData.lambda$generateProject$0(ZigProjectConfigurationData.java:98)
	at com.intellij.openapi.application.WriteAction.lambda$run$1(WriteAction.java:85)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteAction(AnyThreadWriteThreadingSupport.kt:328)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteAction(AnyThreadWriteThreadingSupport.kt:322)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:900)
	at com.intellij.openapi.application.WriteAction.run(WriteAction.java:84)
	at com.falsepattern.zigbrains.project.ide.newproject.ZigProjectConfigurationData.generateProject(ZigProjectConfigurationData.java:85)
	at com.falsepattern.zigbrains.project.ide.newproject.ZigDirectoryProjectGenerator.generateProject(ZigDirectoryProjectGenerator.java:60)
	at com.falsepattern.zigbrains.project.ide.newproject.ZigDirectoryProjectGenerator.generateProject(ZigDirectoryProjectGenerator.java:35)
	at com.intellij.ide.util.projectWizard.AbstractNewProjectStep.doGenerateProject(AbstractNewProjectStep.java:258)
	at com.intellij.ide.util.projectWizard.AbstractNewProjectStep.doGenerateProject(AbstractNewProjectStep.java:193)
	at com.intellij.ide.util.projectWizard.AbstractNewProjectStep$AbstractCallback.consume(AbstractNewProjectStep.java:184)
	at com.intellij.ide.util.projectWizard.ProjectSettingsStepBase$1.actionPerformed(ProjectSettingsStepBase.java:151)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1972)
	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2314)
	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:407)
	at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
	at java.desktop/javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:279)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6662)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3394)
	at java.desktop/java.awt.Component.processEvent(Component.java:6427)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5032)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4860)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4963)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4577)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4518)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2810)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4860)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:783)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:728)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:98)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:755)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:753)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:752)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:696)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.kt:635)
	at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$14(IdeEventQueue.kt:581)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:84)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:581)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:73)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1$1.compute(IdeEventQueue.kt:357)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1$1.compute(IdeEventQueue.kt:356)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:843)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.invoke(IdeEventQueue.kt:356)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.invoke(IdeEventQueue.kt:351)
	at com.intellij.ide.IdeEventQueueKt$performActivity$runnableWithWIL$1.invoke$lambda$0(IdeEventQueue.kt:1035)
	at com.intellij.openapi.application.WriteIntentReadAction.lambda$run$0(WriteIntentReadAction.java:24)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:84)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteIntentReadAction(ApplicationImpl.java:910)
	at com.intellij.openapi.application.WriteIntentReadAction.compute(WriteIntentReadAction.java:55)
	at com.intellij.openapi.application.WriteIntentReadAction.run(WriteIntentReadAction.java:23)
	at com.intellij.ide.IdeEventQueueKt$performActivity$runnableWithWIL$1.invoke(IdeEventQueue.kt:1035)
	at com.intellij.ide.IdeEventQueueKt$performActivity$runnableWithWIL$1.invoke(IdeEventQueue.kt:1035)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:1036)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:114)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1036)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$10(IdeEventQueue.kt:351)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:397)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
	Suppressed: java.lang.RuntimeException: Cannot invoke (class=, method=after, topic=BulkFileListener)
		... 85 more
	Caused by: java.lang.NullPointerException: getSourceRoots(...) must not be null
		at com.intellij.workspaceModel.core.fileIndex.impl.NonIncrementalContributors.computeFileSets(NonIncrementalContributors.kt:160)
		at com.intellij.workspaceModel.core.fileIndex.impl.NonIncrementalContributors.updateIfNeeded(NonIncrementalContributors.kt:50)
		at com.intellij.workspaceModel.core.fileIndex.impl.WorkspaceFileIndexDataImpl.ensureIsUpToDate(WorkspaceFileIndexDataImpl.kt:154)
		at com.intellij.workspaceModel.core.fileIndex.impl.WorkspaceFileIndexDataImpl.getFileInfo(WorkspaceFileIndexDataImpl.kt:94)
		at com.intellij.workspaceModel.core.fileIndex.impl.WorkspaceFileIndexImpl.getFileInfo(WorkspaceFileIndexImpl.kt:260)
		at com.intellij.workspaceModel.core.fileIndex.impl.WorkspaceFileIndexImpl.findFileSet(WorkspaceFileIndexImpl.kt:216)
		at com.intellij.workspaceModel.core.fileIndex.impl.WorkspaceFileIndexImpl.isInContent(WorkspaceFileIndexImpl.kt:82)
		at com.intellij.openapi.roots.impl.ProjectFileIndexImpl.isInContent(ProjectFileIndexImpl.java:209)
		at com.intellij.openapi.roots.impl.PushedFilePropertiesUpdaterImpl.createRecursivePushTask$lambda$11$lambda$10(PushedFilePropertiesUpdaterImpl.kt:187)
		at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runReadAction(AnyThreadWriteThreadingSupport.kt:228)
		at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runReadAction(AnyThreadWriteThreadingSupport.kt:221)
		at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:857)
		at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:66)
		at com.intellij.openapi.roots.impl.PushedFilePropertiesUpdaterImpl.createRecursivePushTask$lambda$11(PushedFilePropertiesUpdaterImpl.kt:187)
		at com.intellij.openapi.roots.impl.PushedFilePropertiesUpdaterImpl$Companion.invokeConcurrentlyIfPossible(PushedFilePropertiesUpdaterImpl.kt:499)
		at com.intellij.openapi.roots.impl.PushedFilePropertiesUpdaterImpl.runConcurrentlyIfPossible(PushedFilePropertiesUpdaterImpl.kt:167)
		at com.intellij.openapi.roots.impl.PushedFilePropertiesUpdaterImpl.processAfterVfsChanges(PushedFilePropertiesUpdaterImpl.kt:146)
		at com.intellij.psi.impl.file.impl.PsiVFSListener$1.after(PsiVFSListener.java:137)
		at com.intellij.util.messages.impl.MessageBusImplKt.invokeMethod(MessageBusImpl.kt:722)
		at com.intellij.util.messages.impl.MessageBusImplKt.invokeListener(MessageBusImpl.kt:686)
		... 84 more
Caused by: java.lang.NullPointerException: getSourceRoots(...) must not be null
	at com.intellij.workspaceModel.core.fileIndex.impl.NonIncrementalContributors.computeFileSets(NonIncrementalContributors.kt:160)
	at com.intellij.workspaceModel.core.fileIndex.impl.NonIncrementalContributors.updateIfNeeded(NonIncrementalContributors.kt:50)
	at com.intellij.workspaceModel.core.fileIndex.impl.WorkspaceFileIndexDataImpl.ensureIsUpToDate(WorkspaceFileIndexDataImpl.kt:154)
	at com.intellij.workspaceModel.core.fileIndex.impl.WorkspaceFileIndexDataImpl.getFileInfo(WorkspaceFileIndexDataImpl.kt:94)
	at com.intellij.workspaceModel.core.fileIndex.impl.WorkspaceFileIndexImpl.getFileInfo(WorkspaceFileIndexImpl.kt:260)
	at com.intellij.openapi.roots.impl.ProjectFileIndexImpl.isInProjectOrExcluded(ProjectFileIndexImpl.java:87)
	at com.intellij.history.integration.LocalHistoryEventDispatcher.createRecursivelyUsingWorkspaceTraversal(LocalHistoryEventDispatcher.kt:55)
	at com.intellij.history.integration.LocalHistoryEventDispatcher.createRecursively(LocalHistoryEventDispatcher.kt:73)
	at com.intellij.history.integration.LocalHistoryEventDispatcher.fileCreated(LocalHistoryEventDispatcher.kt:44)
	at com.intellij.history.integration.LocalHistoryEventDispatcher.handleAfterEvent(LocalHistoryEventDispatcher.kt:179)
	at com.intellij.history.integration.LocalHistoryEventDispatcher.handleAfterEvents$lambda$3(LocalHistoryEventDispatcher.kt:169)
	at com.intellij.history.integration.IdeaGateway.runWithVfsEventsDispatchContext(IdeaGateway.java:156)
	at com.intellij.history.integration.LocalHistoryEventDispatcher.handleAfterEvents(LocalHistoryEventDispatcher.kt:167)
	at com.intellij.history.integration.LocalHistoryEventDispatcher.access$handleAfterEvents(LocalHistoryEventDispatcher.kt:20)
	at com.intellij.history.integration.LocalHistoryEventDispatcher$LocalHistoryBulkFileListener.after(LocalHistoryEventDispatcher.kt:216)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeMethod(MessageBusImpl.kt:722)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeListener(MessageBusImpl.kt:686)
	... 84 more
```

I also tried to create a Hello World project with `zig init` and opening that one from existing sources with the exact same result.

**Comment by FalsePattern** [comment]
This is a known bug, please use this workaround for now: https://github.com/FalsePattern/ZigBrains/issues/60#issuecomment-2290196252

**Comment by v3rm0n** [comment]
I have the same issue on OSX with version 17.1.0-241. I don't have any relative paths in the config, LSP plugin is installed

```
<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="ZLSSettings">
    <option name="zlsPath" value="/Users/maidok/.bin/zls" />
  </component>
  <component name="ZigProjectSettings">
    <option name="toolchainHomeDirectory" value="/Users/maidok/bin/zig-macos-aarch64-0.14.0" />
  </component>
</project>

```

```
java.lang.NullPointerException: getSourceRoots(...) must not be null
	at com.intellij.workspaceModel.core.fileIndex.impl.NonIncrementalContributors.computeFileSets(NonIncrementalContributors.kt:153)
	at com.intellij.workspaceModel.core.fileIndex.impl.NonIncrementalContributors.updateIfNeeded(NonIncrementalContributors.kt:48)
	at com.intellij.workspaceModel.core.fileIndex.impl.WorkspaceFileIndexDataImpl.ensureIsUpToDate(WorkspaceFileIndexDataImpl.kt:147)
	at com.intellij.workspaceModel.core.fileIndex.impl.WorkspaceFileIndexDataImpl.getFileInfo(WorkspaceFileIndexDataImpl.kt:87)
	at com.intellij.workspaceModel.core.fileIndex.impl.WorkspaceFileIndexImpl.getFileInfo(WorkspaceFileIndexImpl.kt:245)
	at com.intellij.workspaceModel.core.fileIndex.impl.WorkspaceFileIndexImpl.findFileSet(WorkspaceFileIndexImpl.kt:201)
	at com.intellij.workspaceModel.core.fileIndex.impl.WorkspaceFileIndexImpl.isInWorkspace(WorkspaceFileIndexImpl.kt:58)
	at com.intellij.util.indexing.roots.IndexableFilesIndexImpl.shouldBeIndexed(IndexableFilesIndexImpl.java:62)
	at com.intellij.util.indexing.FileBasedIndexImpl$5.isInSet(FileBasedIndexImpl.java:288)
	at com.intellij.util.indexing.FileBasedIndexImpl.getContainingProjects(FileBasedIndexImpl.java:1742)
	at com.intellij.util.indexing.FileBasedIndexImpl.lambda$ensureFileBelongsToIndexableFilter$30(FileBasedIndexImpl.java:1918)
	at com.intellij.util.indexing.projectFilter.IncrementalProjectIndexableFilesFilterHolder$ensureFileIdPresent$matchedProjects$2.invoke(ProjectIndexableFilesFilterHolder.kt:90)
	at com.intellij.util.indexing.projectFilter.IncrementalProjectIndexableFilesFilterHolder$ensureFileIdPresent$matchedProjects$2.invoke(ProjectIndexableFilesFilterHolder.kt:90)
	at kotlin.UnsafeLazyImpl.getValue(Lazy.kt:81)
	at com.intellij.util.indexing.projectFilter.IncrementalProjectIndexableFilesFilterHolder.ensureFileIdPresent$lambda$0(ProjectIndexableFilesFilterHolder.kt:90)
	at com.intellij.util.indexing.projectFilter.IncrementalProjectIndexableFilesFilterHolder.access$ensureFileIdPresent$lambda$0(ProjectIndexableFilesFilterHolder.kt:46)
	at com.intellij.util.indexing.projectFilter.IncrementalProjectIndexableFilesFilterHolder$ensureFileIdPresent$1$fileIsInProject$1.invoke(ProjectIndexableFilesFilterHolder.kt:93)
	at com.intellij.util.indexing.projectFilter.IncrementalProjectIndexableFilesFilterHolder$ensureFileIdPresent$1$fileIsInProject$1.invoke(ProjectIndexableFilesFilterHolder.kt:92)
	at com.intellij.util.indexing.projectFilter.IncrementalProjectIndexableFilesFilter$ensureFileIdPresent$1.invoke(IncrementalProjectIndexableFilesFilter.kt:26)
	at com.intellij.util.indexing.projectFilter.IncrementalProjectIndexableFilesFilter$ensureFileIdPresent$1.invoke(IncrementalProjectIndexableFilesFilter.kt:21)
	at com.intellij.util.indexing.projectFilter.ProjectIndexableFilesFilter.runUpdate(ProjectIndexableFilesFilter.kt:35)
	at com.intellij.util.indexing.projectFilter.IncrementalProjectIndexableFilesFilter.ensureFileIdPresent(IncrementalProjectIndexableFilesFilter.kt:21)
	at com.intellij.util.indexing.projectFilter.IncrementalProjectIndexableFilesFilterHolder.ensureFileIdPresent(ProjectIndexableFilesFilterHolder.kt:92)
	at com.intellij.util.indexing.FileBasedIndexImpl.ensureFileBelongsToIndexableFilter(FileBasedIndexImpl.java:1918)
	at com.intellij.util.indexing.FileBasedIndexImpl.scheduleFileForIndexing(FileBasedIndexImpl.java:1830)
	at com.intellij.util.indexing.events.ChangedFilesCollector$4.process(ChangedFilesCollector.java:194)
	at com.intellij.util.indexing.events.ChangedFilesCollector$5.lambda$process$0(ChangedFilesCollector.java:241)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeNonCancelableSection$3(CoreProgressManager.java:269)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:735)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:691)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$computeInNonCancelableSection$4(CoreProgressManager.java:277)
	at com.intellij.openapi.progress.Cancellation.computeInNonCancelableSection(Cancellation.java:57)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeInNonCancelableSection(CoreProgressManager.java:277)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeNonCancelableSection(CoreProgressManager.java:268)
	at com.intellij.util.indexing.events.ChangedFilesCollector$5.lambda$process$1(ChangedFilesCollector.java:240)
	at com.intellij.util.ConcurrencyUtil.withLock(ConcurrencyUtil.java:246)
	at com.intellij.util.indexing.events.ChangedFilesCollector$5.process(ChangedFilesCollector.java:238)
	at com.intellij.util.indexing.events.VfsEventsMerger.processChanges(VfsEventsMerger.java:115)
	at com.intellij.util.indexing.events.ChangedFilesCollector.processFilesInReadAction(ChangedFilesCollector.java:235)
	at com.intellij.util.indexing.events.ChangedFilesCollector.processFilesToUpdateInReadAction(ChangedFilesCollector.java:183)
	at com.intellij.util.indexing.events.ChangedFilesCollector.ensureUpToDate(ChangedFilesCollector.java:144)
	at com.intellij.util.indexing.FileBasedIndexImpl.ensureUpToDate(FileBasedIndexImpl.java:769)
	at com.intellij.util.indexing.FileBasedIndexEx.processExceptions(FileBasedIndexEx.java:296)
	at com.intellij.util.indexing.FileBasedIndexEx.getContainingFilesIterator(FileBasedIndexEx.java:243)
	at com.intellij.psi.search.FileTypeIndex.processFiles(FileTypeIndex.java:52)
	at org.jetbrains.kotlin.idea.base.util.ProjectStructureUtils.containsNonScriptKotlinFile(ProjectStructureUtils.kt:117)
	at org.jetbrains.kotlin.idea.PluginStartupActivity$execute$2.invoke$lambda$0(PluginStartupActivity.kt:28)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$OTelMonitor.callWrapped(NonBlockingReadActionImpl.java:851)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$OTelMonitor$MonitoredComputation.call(NonBlockingReadActionImpl.java:883)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.insideReadAction(NonBlockingReadActionImpl.java:604)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.lambda$attemptComputation$4(NonBlockingReadActionImpl.java:567)
	at com.intellij.openapi.application.impl.RwLockHolder.tryRunReadAction(RwLockHolder.kt:310)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:958)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runInReadActionWithWriteActionPriority$0(ProgressIndicatorUtils.java:93)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtilService.runActionAndCancelBeforeWrite(ProgressIndicatorUtilService.java:66)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runActionAndCancelBeforeWrite(ProgressIndicatorUtils.java:155)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runWithWriteActionPriority$1(ProgressIndicatorUtils.java:138)
	at com.intellij.openapi.progress.ProgressManager.lambda$runProcess$0(ProgressManager.java:100)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:217)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:660)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:735)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:691)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:659)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:79)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:202)
	at com.intellij.openapi.progress.ProgressManager.runProcess(ProgressManager.java:100)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runWithWriteActionPriority(ProgressIndicatorUtils.java:135)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runInReadActionWithWriteActionPriority(ProgressIndicatorUtils.java:93)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.attemptComputation(NonBlockingReadActionImpl.java:567)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.lambda$transferToBgThread$1(NonBlockingReadActionImpl.java:466)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.lambda$transferToBgThread$2(NonBlockingReadActionImpl.java:481)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:840)

```

**Comment by FalsePattern** [comment]
Should be properly fixed in 17.2.0.

**Comment by v3rm0n** [comment]
Works for me, thanks!
